crates/precompile/src/bn128/arkworks.rs

```diff
@@ -0,0 +1,213 @@
+use super::{FQ2_LEN, FQ_LEN, G1_LEN, SCALAR_LEN};
+use crate::PrecompileError;
+use std::vec::Vec; // TODO: Remove this import, `cargo clippy --no-default-features` needs it to disambiguate
+
```

Comment:
I haven't investigated this error, though its happening on another PR too

Reply:
This is fine to leave it as is (If you mean `use std::vec::Vec`)